.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -125,7 +125,7 @@ jobs:
           cp -rf ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
       - name: Install dependencies
-        run: python -m pip install cibuildwheel~=2.16.0 toml
+        run: python -m pip install cibuildwheel~=2.16.0 tomlkit
```

Comment:
> I'm wondering if it works with arm64?

@multiphaseCFD yes, it works on arm64; the behaviour should be the same as both tomlkit and tomli/tomllib have consistent supports on arm64 and x86 arch